app/views/admin/cms/collections/_form.html.erb

```diff
@@ -32,6 +34,13 @@
   </div>
 
   <div>
+    <%= f.label :image, "Default Collection Image" %>
+    <%= f.file_field :image %>
+  </div>
+
+  <img src="<%= @collection.image_url %>">
```

Comment:
I've changed this code to only display the image element if the collection has an image associated 👍